### ISSUE TITLE
Modify managed nodegroups launch template template tags_specifications

### ISSUE THREAD

**Issue by YakobovLior** [terraform-aws-modules/terraform-aws-eks#1689]
## Versions

 - Terragrunt: v0.31.0
 - Terraform: v1.0.2

## Description

I'm looking for the ability to configure tags_specifications for the created launch template, without the need to create a dedicated launch template in advance. 

```
module "eks" {
  source          = "terraform-aws-modules/eks/aws"
  cluster_name    = "${local.envname}-eks-cluster"
  cluster_version = "1.21"
  vpc_id  = var.vpc_id
  subnets = var.private_subnet_ids  

  cluster_create_timeout                         = "1h"
  cluster_endpoint_private_access                = true
  cluster_endpoint_public_access                 = true
  cluster_create_endpoint_private_access_sg_rule = true
  cluster_endpoint_private_access_cidrs          = var.ingress_cidr_blocks

  node_groups_defaults = {
    ami_type  = "AL2_x86_64"
    subnets   = ["subnet-0825..."]
  }

  node_groups = {
    # Nodegroup for main pods (jenkins master, gitlab services, etc)
    gnrl = {
      desired_capacity = 2
      min_capacity     = 2
      max_capacity     = 2

      instance_types = [var.instance_type_main_ng]
      capacity_type = "ON_DEMAND"
      disk_size     = var.disk_size_main_ng
      k8s_labels = {
        Environment = var.environment
        "node-role.eks.io/main" = "true"
      }
    }

    # Nodegroup for jenkins workers pods
    jwork = {
      desired_capacity = 2
      min_capacity     = 2
      max_capacity     = 10

      instance_types = [var.instance_type_jenkins_workers_ng]
      capacity_type = "SPOT"
      disk_size     = var.disk_size_jenkins_workers_ng
      k8s_labels = {
        Environment = var.environment
        "node-role.eks.io/jenkins-workers" = "true"
      }
    }
  }
```

Eventually, 2 nodegroups have been created but since there is no template tags with key **Name**,

![Screen Shot 2021-11-10 at 17 11 56](https://user-images.githubusercontent.com/33580144/141139322-ff016b39-0458-47bf-b04d-48bca0e252c8.png)

My instances are being created with any names at all:

![Screen Shot 2021-11-10 at 17 12 21](https://user-images.githubusercontent.com/33580144/141139379-a414a36b-ad20-432b-ad56-a90d902a7a26.png)

Is there a way to achieve my goal?
Thanks,

Lior

**Comment by daroga0002** [comment]
> Is there a way to achieve my goal?

not by this module as EKS node group API doesnt provide such option.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 18.0.0 :tada:
